algebird-test/src/test/scala/com/twitter/algebird/SpaceSaverTest.scala

```diff
@@ -26,22 +31,31 @@ class SpaceSaverLaws extends CheckProperties {
           Equiv.fromFunction { (left, right) =>
             (left consistentWith right) && (right consistentWith left)
           }
+
         commutativeSemigroupLaws[SpaceSaver[Int]]
       }
     }
   }
+
+  property("SpaceSaver can serialize/deserialize itself") {
+    forAll { ss: SpaceSaver[String] =>
```

Comment:
Pending this one change this looks great to me! @ElPicador, thanks for responding to these rounds of feedback. With this extra test the new serde code will have 100% coverage :)